start_cerberus.py

```diff
@@ -227,6 +230,25 @@ def main(cfg):
             if cerberus_publish_status:
                 publish_cerberus_status(cerberus_status)
 
+            # Alert on high latencies
+            # Intialize prometheus client
+            if distribution == "openshift" and not prometheus_url:
+                url = runcommand.invoke(r"""oc get routes -n openshift-monitoring -o=jsonpath='{.items[?(@.metadata.name=="prometheus-k8s")].spec.host}'""") # noqa
+                prometheus_url = "https://" + url
+            if distribution == "openshift" and not prometheus_bearer_token:
+                prometheus_bearer_token = runcommand.invoke("oc -n openshift-monitoring sa get-token prometheus-k8s") # noqa
+            if prometheus_url and prometheus_bearer_token:
+                promcli.initialize_prom_client(prometheus_url, prometheus_bearer_token)
+                # Check for high latency alerts
+                query = r"""ALERTS{alertname="KubeAPILatencyHigh", severity="warning"}"""
+                metrics = promcli.get_metrics(query)
+                if len(metrics) != 0:
+                    logging.warning("Kubernetes API server latency is high. "
+                                     "More than 99th percentile latency for given requests to the kube-apiserver is above 1 second.\n") # noqa
+                    logging.info(metrics)
```

Comment:
Maybe newline can be added after logging metrics to ensure report is neat. `logging.info("%s\n" % metrics)`

Reply:
Fixed it, thanks.